contrib/templates/data-engineering/base/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,50 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+include:
+  - assets/*.yml
+  - assets/*/*.yml
+
+variables:
+  catalog:
+    description: The catalog to use
```

Comment:
we could add a `default: main` here

Reply:
I always found that style a bit harder to grok for new users. It seems a bit easier if they can just edit the actual values for each target lower down and don't need to think about defaults vs. overrides.